### PR TITLE
Added Checklist and Known Issues section

### PR DESCRIPTION
**Pull request by edyuenyw**
Added initial changes for https://digital-engagement.atlassian.net/browse/SDPA-5804

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,22 @@
 
 https://dpc-sdp.github.io/sdp-docs/
 
+
```
Comment (anthony-malkoun) [inline README.md:6]: As discussed @edyuenyw `README.md` is about setting up the `dpc-sdp/sdp-docs` project, not about setting up the reference site. Everything under the `docs` directory in this project gets deployed (eventually) to https://dpc-sdp.github.io/sdp-docs/ so your changes should live under there. Thinking about this I'm thinking a new page under the `docs/working-with-sdp.md` for anything DPC/Cenitex specific and add to `docs/working-with-sdp.md` for other steps. Thoughts?

**Review by anthony-malkoun — CHANGES_REQUESTED** [review]
Hi @edyuenyw see my comments about getting these changes into the `docs` directory.

File: README.md (hunk 1 above)
Comment (edyuenyw) [reply README.md]: @anthony-malkoun I've made further changes and add more details to my commit. I also "reverted" the README.md commit which I still can see it in my log though there was not changed made to README.md anymore.
Can you please check and let me know if I need to change anything else?

**Review by anthony-malkoun — CHANGES_REQUESTED** [review]
Hi @edyuenyw can you update your build to use pygmy-go and document? I think there's value in creating a subpage on this for DPC/Cenitex with issues. You can create in this PR or create a new one if you prefer

File: mkdocs.yml [hunk 2]
```diff
@@ -97,6 +97,7 @@ markdown_extensions:
 nav:
   - About: index.md
   - Working with SDP: working-with-sdp.md
+  - Troubleshooting: troubleshooting.md
```
Comment (anthony-malkoun) [inline mkdocs.yml:100]: I think this should be a subpage of the `working-with-sdp.md` page. Just indent (see Bay: below)

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Troubleshooting
+
+!!! info
+    
+    You are viewing a version of the documentation that is currently under 
+    active development.
+
+ This page provides some workaround for issues that you may encounter during a local build.
```
Comment (anthony-malkoun) [inline docs/troubleshooting.md:8]: *workarounds

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Open Workspace | Cenitex | Citrix
+Unable to install software (e.g Docker Desktop) or run commands which requires elevated permissions.
```
Comment (anthony-malkoun) [inline docs/troubleshooting.md:11]: *require

File: docs/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Check if you Open Workspace is in your machine. This will allow software installation that requires elevated permissions.
+2. If you believe to have Open Workspace installed but still have issue with installations, logoff from Citrix Gateway (Windows Taskbar Icon) and reconnect through Go Connect (Window search). An IE browser will open and ask for your DPC login credentials. Process might take some time to complete for others and for some, browser seemed to be hanging. In that case try restarting your computer and retry Go Connect again.
+3. Check again to see if you have the elevated permissions or not. You might have restart your computer several times to take effect.
```
Comment (anthony-malkoun) [inline docs/troubleshooting.md:15]: *have to restart

File: docs/troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+## SSH Key
```
Comment (anthony-malkoun) [inline docs/troubleshooting.md:17]: Delete if no issues.

File: docs/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+*(add ssh key issue if any)*
+
+## Lagoon | Lagoon CLI
+*(Remote work)* Unable to resolve host or cannot descrypt encoded SSH
```
Comment (anthony-malkoun) [inline docs/troubleshooting.md:21]: *decrypt

File: docs/working-with-sdp.md [hunk 8]
```diff
@@ -26,12 +26,71 @@ with the DPC team there are a few tasks to complete.
     ticket.
     - There are GitHub repositories for both the front-end and back-end applications that you will be working on.
 
-## Setup key in the Lagoon Dashboard
+## Install dependencies
 
-Once you have access to the [Lagoon Dashboard](https://dashboard.amazeeio.cloud/projects/) make sure to add your
+### Create SSH key pair
+
+- [Generate new SSH key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) will guide you on how to generate, test and also add your key to your GitHub [*(Add a new SSH key)*](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)
```
Comment (anthony-malkoun) [inline docs/working-with-sdp.md:33]: Did the instruction specify a different key type then `id_rsa`? If so don't we need to make a note to only use `id_rsa` and not the suggested one?

File: docs/working-with-sdp.md [hunk 9]
```diff
@@ -26,12 +26,71 @@ with the DPC team there are a few tasks to complete.
...
+            - HostPort: 8080
+            443/tcp:
+            - HostPort: 443
+    ```
+
+### Setup Docker
+- Download and Install Docker Desktop (https://www.docker.com/get-started).
+- After installation, run:
+
+    ```
```
Comment (anthony-malkoun) [inline docs/working-with-sdp.md:84]: Please explain the differences between `ahoy build` (destroys all containers and rebuilds from scratch) and `ahoy up` which just starts existing containers.

File: docs/working-with-sdp.md [hunk 10]
```diff
@@ -26,12 +26,71 @@ with the DPC team there are a few tasks to complete.
...
+### Setup Pygmy-Go
+
+- Drop-in replacement using latest [release](https://github.com/fubarhouse/pygmy-go/releases/tag/v0.7.1)
+- After installation, run:
+
+    ```
+    pygmy-go up
+    ```
+
+- Create a new .pygmy.yml file so haproxy can be configured to another port other than 80
```
Comment (anthony-malkoun) [inline docs/working-with-sdp.md:66]: This should go into the troubleshooting document IMO as it's a Cenitex only issue (as it hogs port 80)

File: docs/troubleshooting.md [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+    
+    You are viewing a version of the documentation that is currently under 
+    active development.
+
+ This page provides some workaround for issues that you may encounter during a local build.
+
+## Open Workspace | Cenitex | Citrix
+Unable to install software (e.g Docker Desktop) or run commands which requires elevated permissions.
+
+1. Check if you Open Workspace is in your machine. This will allow software installation that requires elevated permissions.
```
Comment (anthony-malkoun) [inline docs/troubleshooting.md:13]: *Check if Open Workplace is installed on your machine.

Also, a screen shot of the PCInfo app would help here. Ping me if you need help with this.

File: docs/troubleshooting.md (hunk 5 above)
Comment (anthony-malkoun) [reply docs/troubleshooting.md:15]: *several times for it it to take effect.

File: docs/troubleshooting.md [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+2. If you believe to have Open Workspace installed but still have issue with installations, logoff from Citrix Gateway (Windows Taskbar Icon) and reconnect through Go Connect (Window search). An IE browser will open and ask for your DPC login credentials. Process might take some time to complete for others and for some, browser seemed to be hanging. In that case try restarting your computer and retry Go Connect again.
```
Comment (anthony-malkoun) [inline docs/troubleshooting.md:14]:
```suggestion
2. If you believe Open Workplace is installed but are still having issues with installations, logoff from Citrix Gateway (Windows Taskbar Icon) and reconnect through Go Connect (Window search). An IE browser will open and ask for your DPC login credentials. Process might take some time to complete or your browser might appear to hang. In that case try restarting your computer and retry Go Connect.
```

**Review by anthony-malkoun — CHANGES_REQUESTED** [review]
Hi @edyuenyw most of the changes are small grammar/spelling issues. There are a few bigger changes I'd like to see. Let me know if you need more detail.

**Review by anthony-malkoun — APPROVED** [review]
Thanks for making the requested changes @edyuenyw. Please feel free to merge this in when you get a chance.

**Pull request merged**